miner/worker.go

```diff
@@ -1113,7 +1119,11 @@ loop:
 				circuitCapacityReached = false
 
 				// Store skipped transaction in local db
-				rawdb.WriteSkippedTransaction(w.eth.ChainDb(), tx, "row consumption overflow", w.current.header.Number.Uint64(), nil)
+				if w.config.StoreSkippedTxTraces {
+					rawdb.WriteSkippedTransaction(w.eth.ChainDb(), tx, traces, "row consumption overflow", w.current.header.Number.Uint64(), nil)
+				} else {
+					rawdb.WriteSkippedTransaction(w.eth.ChainDb(), tx, nil, "row consumption overflow", w.current.header.Number.Uint64(), nil)
+				}
```

Comment:
var skippedTraces *types.BlockTrace
```suggestion
var skippedTraces *types.BlockTrace
    if w.config.StoreSkippedTxTraces {
       skippedTraces = traces
    }
    rawdb.WriteSkippedTransaction(w.eth.ChainDb(), tx, skippedTraces, "row consumption overflow", w.current.header.Number.Uint64(), nil)
```